doc/running_local_cluster.md

```diff
@@ -12,20 +19,18 @@ Using a local cluster you can test/play/interact with all the Cardano blockchain
 - virtualenv
 - nix
 
-You can find here how to install `nix` and how to add **IOHK** binary cache:
-<https://github.com/input-output-hk/cardano-node/blob/master/doc/getting-started/building-the-node-using-nix.md>
-
+You can find [here](<https://github.com/input-output-hk/cardano-node/blob/master/doc/getting-started/building-the-node-using-nix.md>) how to install `nix` and how to add **IOHK** binary cache
```

Comment:
Could be helpfull to point that the framework doesn't work on windows

Reply:
This would be only for power users - we don't want to give a lot of details especially for avoiding future questions.